File: Arcade/Intro/04_adjacentElementsProduct.js
function adjacentElementsProduct(inputArray) {
    var max = inputArray[0] * inputArray[1];
    for (var i = 2; i < inputArray.length; i++) {
        if (inputArray[i-1] * inputArray[i] > max) {
            max = inputArray[i-1] * inputArray[i];
        }
    } return max;
}

File: Arcade/Intro/C++/04_adjacentElementsProduct.cpp
int adjacentElementsProduct(std::vector<int> inputArray) {
    int highest=INT_MIN;
        for(int i=0;i<inputArray.size()-1;i++){
                int test=inputArray[i]*inputArray[i+1];
                if(test>highest) highest = test;
        }
        return highest;
}
